lab_1/main.py

```diff
@@ -25,7 +32,25 @@ def remove_stop_words(tokens: list, stop_words: list) -> list:
     stop_words = ['the', 'is']
     --> ['weather', 'sunny', 'man', 'happy']
     """
-    pass
+    flag = 'ok'
```

Comment:
This type of check is interesting, but try to use built-in Python ways, consider `return`.